website/pages/docs/platform/k8s/helm/configuration.mdx

```diff
@@ -280,6 +306,9 @@ and consider if they're appropriate for your deployment.
       - `path` (`string: /vault/userconfigs`) -
         Name of the path where a configMap or secret is mounted. If not specified
         the volume will be mounted to `/vault/userconfig/<name of volume>`.
+      
+      - `defaultMode` (`string: 420) - 
```

Comment:
```suggestion
      - `defaultMode` (`string: 420`) - 
```

Not sure if this should be `(string: "420")` or not, because it's a string 🤔

Reply:
I put quotes around it, thanks!